core/src/main/java/org/apache/accumulo/core/metrics/MetricsUtil.java

```diff
@@ -78,7 +79,7 @@ private static void initializeMetrics(boolean enabled, boolean jvmMetricsEnabled
 
       List<Tag> tags = new ArrayList<>();
       tags.add(Tag.of("process.name", processName));
-
+      tags.add(Tag.of("resource.group", resourceGroup));
```

Comment:
wondering if we should add instance_name to the tags...

Reply:
If I recall correctly, there are tags that are automatically added on initialization - maybe we could make that a default?  I'm not sure of the impact of adding a "constant" tag (assuming most installations run a single instance). Or maybe instance name is synonymous with system name?

If metrics are forwarded to a central collector then we should support a tag that allows differentiation, so the usages may be wider than running multiple instances within a single cluster,

Reply:
Likely outside of the scope of this PR, but maybe we could consider migrating the standard tags to something that mirrors the syslog naming conventions (https://datatracker.ietf.org/doc/html/rfc5424)

That could make it easier to correlate metrics and log messages that are sent to syslog.

Reply:
added instance name tag in #3998 targeting 2.1.3